static/src/javascripts/projects/common/modules/commercial/creatives/foundation-funded-logo.js

```diff
@@ -18,22 +20,19 @@ define([
      * * https://www.google.com/dfp/59666047#delivery/CreateCreativeTemplate/creativeTemplateId=10022127
      * * https://www.google.com/dfp/59666047#delivery/CreateCreativeTemplate/creativeTemplateId=10026327
      */
-    var Template = function ($adSlot, params) {
-        this.$adSlot       = $adSlot;
+    var Template = function (adSlot, params) {
```

Comment:
This seems an odd name to me - normally I'd name the default export of a module after the module itself. And I'd typically only use the term 'template' to refer to the actual view template.